examples/permissioned-token/test/failing-paths.js

```diff
@@ -0,0 +1,178 @@
+  });
+
+  it('WhiteListing', () => {
+    // Only asset manager can whitelist
+    const address = elon.address;
+    optInToPermissions(runtime, address, permissionsAppId);
+    syncAccounts();
+    assert.isDefined(elon.getAppFromLocal(permissionsAppId));
+
+    // Fails because Bob is not the manager
+    assert.throws(() => runtime.executeTx({
+      type: types.TransactionType.CallNoOpSSC,
+      sign: types.SignType.SecretKey,
+      fromAccount: bob.account, // Bob is not the asset manager
+      appId: permissionsAppId,
+      payFlags: { totalFee: 1000 },
+      appArgs: ['str:add_whitelist', `int:${controllerAppID}`],
+      accounts: [address],
+      foreignAssets: [assetIndex],
+      foreignApps: [controllerAppID]
+    }), 'RUNTIME_ERR1009');
+  });
+
+  it('Opt Out', () => {
+    // Opt-out wont work without opting in
+    assert.throws(() => optOut(runtime, alice, elon, assetIndex), 'RUNTIME_ERR1404');
+  });
+
+  it('Change Permissions SSC Manager', () => {
+    // Optin In new SSC manager to the SSC
+    optInToPermissions(runtime, elon.address, permissionsAppId);
+    syncAccounts();
+
+    // whitelist operation can only be performed by permissions SSC manager
+    assert.throws(() => whitelist(runtime, elon, bob.address, assetIndex, controllerAppID, permissionsAppId), 'RUNTIME_ERR1009');
+
+    const txn = {
+      type: types.TransactionType.CallNoOpSSC,
+      sign: types.SignType.SecretKey,
+      fromAccount: bob.account, // asset manager account (fails otherwise)
+      appId: permissionsAppId,
+      payFlags: { totalFee: 1000 },
+      appArgs: ['str:change_permissions_manager'],
+      accounts: [elon.address]
+    };
+
+    // fails as fromAccount is not the current SSC manager
+    assert.throws(() => runtime.executeTx(txn), 'RUNTIME_ERR1009');
+  });
+
+  it('Force Transfer', () => {
+    // Opt-In to permissions SSC & Whitelist
+    whitelist(runtime, alice, alice.address, assetIndex, controllerAppID, permissionsAppId);
+    whitelist(runtime, alice, elon.address, assetIndex, controllerAppID, permissionsAppId);
+    whitelist(runtime, alice, bob.address, assetIndex, controllerAppID, permissionsAppId);
+    syncAccounts();
+
+    // Opt-In to ASA
+    optInToASA(runtime, bob.address, assetIndex);
+    optInToASA(runtime, elon.address, assetIndex);
+    syncAccounts();
+
+    // Issue some tokens to sender
+    issue(runtime, alice, bob, 150, controllerAppID, assetIndex, lsig);
+    syncAccounts();
+
+    // Fails as only manager can perform force transfer
+    assert.throws(() =>
+      forceTransfer(runtime, bob, elon, 20, assetIndex, controllerAppID, permissionsAppId, lsig, bob),
+    'RUNTIME_ERR1009'
+    );
+
+    // Fails as reciever will have more than 100 tokens post succesfull transaction
+    assert.throws(() =>
+      forceTransfer(runtime, bob, elon, 101, assetIndex, controllerAppID, permissionsAppId, lsig, alice),
+    'RUNTIME_ERR1009'
+    );
+  });
+
+  it('Token Transfer', () => {
```

Comment:
Headings/titles should be in `describe(..)`. `it(..)` should explain the specific test we are executing here